doc/opt_aero.rst

```diff
@@ -93,20 +93,42 @@ We can set up constraints on the geometry with the DVConstraints class, also fou
 There are several built-in constraint functions within the DVConstraints class, including thickness, surface area, volume, location, and general linear constraints.
 The majority of the constraints are defined based on a triangulated surface representation of the wing obtained from ADflow.
 
-The volume and thickness constraints are set up by creating a 2D grid of points which is projected through the planform of the wing (the points must lie completely inside the wing).
-For the volume constraint, the 2D grid is transformed into a 3D grid bounded by the surface of the wing.
-The volume is computed by adding up the volumes of the cells that make up the 3D grid.
-For the thickness constraints, the nodes of the 2D grid are projected to the upper and lower surface of the wing
-The thickness for a given node is the difference between its upper and lower projections.
+The volume and thickness constraints are set up by creating a uniformly spaced 2D grid of points, which is then projected onto the upper and lower surface of a triangulated-surface representation of the wing.
+The points are projected normal to the chordwise lines connecting the leading and trailing edge points of this grid.
```

Comment:
This sentence is not very accurate in 3D. You can only compute a normal to a surface, not the chord-wise lines. The normal is computed by DVCon automatically if you have a 2D array of thickness constraints (i.e. https://github.com/mdolab/pygeo/blob/master/pygeo/DVConstraints.py#L442). If you add a 1D array of thickness constraints (i.e. https://github.com/mdolab/pygeo/blob/master/pygeo/DVConstraints.py#L600), then you need to specify the projection axis.

Reply:
Added a note clarifying based on comments in the link.